specification/gedcom.md

```diff
@@ -2831,10 +2831,21 @@ A [registry of component subtags](https://www.iana.org/assignments/language-subt
 
 #### `LANG` (Language) `g7:HEAD-LANG`
 
-The language in which the `Text`-typed payloads of all structures in the document appear, except as superseded by a `g7:LANG`.
+The language in which the `Text`-typed payloads of all structures in the dataset appear, except as superseded by a `g7:LANG`.
```

Comment:
I agree with Albert's technical points raised during the last meeting, in that I don't believe this sentence is implementable in general.   A user can type in text from some language and the program cannot guarantee it's in the language the program thinks it is, without actually asking the user (who may then give the wrong answer).